src/settings.plugin.js

```diff
@@ -321,26 +334,51 @@ wikiaJWPlayerSettingsPlugin.prototype.getSuitableCaptionsIndex = function(userLa
 		.indexOf(userLang);
 };
 
-wikiaJWPlayerSettingsPlugin.prototype.captionLangMap = {
-	en: 'English',
-	pl: 'Polski',
-	fr: 'Français',
-	de: 'Deutsch',
-	it: 'Italiano',
-	ja: '日本語',
-	pt: 'Português',
-	ru: 'Русский язык',
-	es: 'Español',
-	zh: '中文'
+wikiaJWPlayerSettingsPlugin.prototype.updateCurrentCaptions = function (track) {
+	for (var i = 0; i < this.captionsList.childNodes.length; i++) {
+		this.captionsList.childNodes[i].classList.remove(isActiveClass);
+	}
+
+	this.captionsList.querySelector('[data-track="' + track + '"]').classList.add(isActiveClass);
 };
 
-wikiaJWPlayerSettingsPlugin.prototype.getLabelElement = function (wrapper) {
-	return wrapper.querySelector('label');
+/**
+ * Hack warning!
```

Comment:
Can you include in doc why is it needed?

Reply:
I changed the docs.